training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/predict.py

```diff
@@ -4,19 +4,22 @@
 """Huggingface predict file for whisper mlflow model."""
 
 import os
-import re
 import base64
+import wget
```

Comment:
Does wget need to be added to the conda yaml of this model?

Reply:
yes. It has to be installed. @novaturient95 could you add wget==3.2 to the conda.yaml pip dependencies